src/test/objectstore/store_test.cc

```diff
@@ -6839,7 +6838,6 @@ INSTANTIATE_TEST_SUITE_P(
   StoreTest,
   ::testing::Values(
     "memstore",
-    "filestore",
```

Comment:
This change requires shifting of --gtest_filter parametes in 
qa/suites/rados/objectstore/backends/* files where ceph_test_objectstore is called.
For example `--gtest_filter=*/2:-*SyntheticMatrixC*` no longer means bluestore but kstore.

Reply:
Done